website/templates/project/nodes_privacy.mako

```diff
@@ -35,7 +35,7 @@
                             </div>
                         </div>
                         <div class="osf-treebeard">
-                            <div id="nodesPrivacyTreebeard-anchor">
+                            <div id="nodesPrivacyTreebeard">
```

Comment:
what's this?

Reply:
https://centerforopenscience.slack.com/archives/C03T0PPJKRS/p1676601726137299 I thought I'd test this and see if we can revert this behavior quickly.

Reply:
merge latest develop to get rid of this delta, please